tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -324,6 +328,20 @@ export const uploadResource = async (args: uploadResourceArgs): Promise<void> =>
   }
 }
 
+export const dropUploadFiles = async (args: uploadResourceArgs): Promise<void> => {
+  const { page, resources } = args
+
+  await dragDropFiles(page, resources, filesView)
+
+  await page.waitForSelector(uploadInfoSuccessLabelSelector)
+  await page.locator(uploadInfoShowDetailsButton).click()
+  await Promise.all(
+    resources.map((file) =>
+      page.waitForSelector(util.format(resourceNameSelector, path.basename(file.name)))
```

Comment:
if you go upload info maybe you should check particular item

<img width="1314" alt="Screenshot 2023-05-22 at 10 58 38" src="https://github.com/owncloud/web/assets/84779829/38984da3-8152-493a-962a-b8f21c7e1df3">

Reply:
The selector is not strict. I will fix it